packages/test/test-end-to-end-tests/src/test/batchBreakRegression.spec.ts

```diff
@@ -254,7 +255,10 @@ describeNoCompat("Batching failures", (getTestObjectProvider) => {
 
         itExpects("force nack",
         [
-            { eventName: "fluid:telemetry:Container:ContainerClose", error: "0x29a" },
+            {
+                eventName: "fluid:telemetry:Container:ContainerClose",
+                error: "Received a system message during batch processing",
```

Comment:
This will cause trouble after tagging. It's probably outside of scope of this task, but here is how it was solved in devmain:

export const tagForOperationX = RESERVETAG('0000');

RESERVETAG was a keyword that tagging script recognized (among many other things like AssertTag) that have-to-have unique tags. So, it would change it on the next pass to some unique value. But given that this var was used both in shipping code and in UTs, it would not break UTs.

Reply:
So if we remove an assert tag, it then becomes available for recycling? Sounds like I should open a task for that..

Reply:
Opened ADO:1992.